zap/src/test/java/org/zaproxy/zap/authentication/AuthenticationMethodIndicatorsUnitTest.java

```diff
@@ -64,6 +65,7 @@ public void setUp() throws Exception {
         header.setURI(new URI("http://www.example.com", true));
         loginMessage.setRequestHeader(header);
         method = Mockito.mock(AuthenticationMethod.class, Mockito.CALLS_REAL_METHODS);
+        method.setAuthCheckingStrategy(AuthCheckingStrategy.EACH_RESP);
```

Comment:
Techincally the `AuthenticationMethod` should properly handle a null strategy as it allows to set it to null.